### ISSUE TITLE
Fix .tigrc for tig 2.x

### ISSUE THREAD

**Issue by tobynet** [tobynet/dotfiles#9]
```shell
$ tig
tig warning: ~/.tigrc:4: show-rev-graph is obsolete; see tigrc(5) for how to set the commit-title column option
tig warning: ~/.tigrc:6: show-refs is obsolete; see tigrc(5) for how to set the commit-title column option
tig warning: ~/.tigrc:7: show-line-numbers is obsolete; see tigrc(5) for how to set the line-number column option
tig warning: ~/.tigrc:9: line-number-interval is obsolete; see tigrc(5) for how to set the line-number column option
tig warning: ~/.tigrc:10: Unknown option name: commit-encoding
tig warning: Errors while loading /home/****/.tigrc.
tig: Not a git repository
```

**Comment by weimengxi** [comment]
I've this trouble, 2

**Comment by tobynet** [comment]
Fix it (572eabe, de5e6f0)
